### ISSUE TITLE
Wells Fargo MFA reset happens very (too) often

### ISSUE THREAD

**Issue by pgorsira** [plaid/support#96]
This may be on WF's end, as lately I've had to do MFA everytime I've logged in through the web interface.

I go through the usual process of onboarding a user to auth: POST username and password, get access token and MFA question. Answer MFA question. No errors from plaid at this point. Then, immediately after, hitting /auth/get gives 

{
"code": 1215,
  "message": "mfa reset",
  "resolve": "MFA access has changed or this application's access has been revoked. Submit a PATCH call to resolve.",
"access_token": "xxx"
}

This was all through my application, so I decided to manually create the PATCH call. I did, it worked (had to MFA again but it went through). I was even able to hit /auth/get once. However, hitting it again in ~1 minute gave the same error as above (requesting MFA patching).

**Comment by pgorsira** [comment]
Also you may want to update https://plaid.com/docs/?api-overview/#institutions and note that Wells Fargo has MFA now.

**Comment by ctrembl** [comment]
Hey @pgorsira - this was in fact the result of a recent update by Wells Fargo. Though we don't have this in the documentation yet (but will add shortly!) and as you have seen, we do support question-based MFA at Wells.

In this particular case, you may need to run a few PATCH calls to make sure all of the information has been updated appropriately. Let us know if you have any other questions!

**Comment by pgorsira** [comment]
Thanks Carl. 

I might be tripping but today I did some testing and I think this issue may actually be related to the `/balance` endpoint. 

My flow: I start by authing a user. I give the MFA response. I upgrade to connect. Both `/connect/get` and `/auth/get` work consistently after this.

I try to hit `/balance`. I get the message "MFA access has changed [...] Submit a PATCH call to resolve." Attempting to hit `/connect/get` and `/auth/get` from this point on gives the same MFA response. 

From this standpoint it seems like something weird is going on on your end when I hit the`/balance` endpoint. I tested this manually today and it seems to check out.
